specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/operations.json

```diff
@@ -0,0 +1,103 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "HDInsightManagementClient",
+		"description": "The HDInsight Management Client.",
+		"version": "2015-03-01-preview"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"consumes": ["application/json"],
+	"produces": ["application/json"],
+	"security": [{
+		"azure_auth": ["user_impersonation"]
+	}],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+	"paths": {
+		"/providers/Microsoft.HDInsight/operations": {
+			"get": {
+				"tags": ["Operations"],
+				"description": "Lists all of the available HDInsight REST API operations.",
+				"operationId": "Operations_List",
+				"parameters": [{
+					"$ref": "#/parameters/ApiVersionParameter"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK. The request has succeeded.",
+						"schema": {
+							"$ref": "#/definitions/OperationListResult"
+						}
+					}
+				},
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				}
+			}
+		}
+	},
+	"definitions": {
```

Comment:
all the properties should be marked as `"readOnly": true`

Reply:
Done.